contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/client/UserFeignClient.java

```diff
@@ -36,4 +37,22 @@ public interface UserFeignClient {
             value = "/verify/{userId}/{token}"
     )
     Boolean verifyUser(@PathVariable("userId") Integer userId, @PathVariable("token") String token);
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/verify/{userId}"
```

Comment:
Let's try to do this in a kind of restful manner

POST @ /tokens/verification
And pass the userId in the body